india_compliance/gst_india/api_classes/e_invoice.py

```diff
@@ -30,6 +30,9 @@ def setup(self, doc=None, *, company_gstin=None):
         if not self.settings.enable_e_invoice:
             frappe.throw(_("Please enable e-Invoicing in GST Settings first"))
 
+        if frappe.utils.scheduler.is_scheduler_disabled():
+            frappe.throw(_("Please enable Scheduler to use e-Invoicing features"))
```

Comment:
Documentation link to enable scheduler.
https://frappeframework.com/docs/user/en/bench/resources/bench-commands-cheatsheet#scheduler

This is a user-facing error, where the user may not understand what the scheduler is.
Add extra comment: Please get in touch with your server administrator to resolve this issue.